framework/source/class/qx/core/Property.js

```diff
@@ -797,30 +811,27 @@ qx.Bootstrap.define("qx.core.Property",
      * Installs a getter into the class prototype, without executing it
      * Supported variants: get
      *
-     * @param instance {Object} the instance which calls the method
      * @param clazz {Class} the class which originally defined the property
      * @param name {String} name of the property
      * @param variant {String} Method variant.
      */
-    installOptimizedGetter : function(instance, clazz, name, variant)
+    installOptimizedGetter : function(clazz, name, variant)
```

Comment:
If the function has never been in a release, then I would say it is fine to change the api